js/apps/account-ui/src/api/methods.ts

```diff
@@ -29,6 +29,13 @@ export async function getPersonalInfo({
   return parseResponse<UserRepresentation>(response);
 }
 
+export async function supportedLocales({ signal }: CallOptions = {}): Promise<
```

Comment:
This should match the naming conventions of the other methods in this file.
```suggestion
export async function getSupportedLocales({ signal }: CallOptions = {}): Promise<
```